File: 240609/소수 구하기/get-prime.py
N = int(input())

def is_prime_number(n):
    for i in range(2, int(n ** 0.5) + 1):
        if n % i == 0:
            return False

    return True

prime_numbers = []
for i in range(2, N + 1):
    if is_prime_number(i):
        prime_numbers.append(i)

print(*prime_numbers)

File: 251110/소수 구하기/get-prime.cpp
#include <iostream>
#include <vector>
#include <cmath>
using namespace std;

bool is_prime_number(int n) 
{
    if (n < 2) 
        return false;
    for (int i = 2; i <= sqrt(n); i++) 
    {
        if (n % i == 0)
            return false;
    }
    return true;
}

int main() 
{
    int N;
    cin >> N;

    vector<int> prime_numbers;
    for (int i = 2; i <= N; i++) 
    {
        if (is_prime_number(i))
            prime_numbers.push_back(i);
    }

    for (size_t i = 0; i < prime_numbers.size(); i++) 
    {
        cout << prime_numbers[i];
        if (i + 1 < prime_numbers.size()) 
            cout << ' ';
    }
    cout << '\n';

    return 0;
}
